docs/source/en/using-diffusers/controlling_generation.mdx

```diff
@@ -0,0 +1,66 @@
+<!--Copyright 2022 The HuggingFace Team. All rights reserved.
```

Comment:
```suggestion
<!--Copyright 2023 The HuggingFace Team. All rights reserved.
```

Reply:
It's fine we can update those all at once :-)